kola/tests/exampleTest

```diff
@@ -0,0 +1,46 @@
+// For each group of related tests a package can be created with
+// necessary helper functions.
+package example
+
+import "github.com/coreos/mantle/harness"
+
+// This is necessary somewhere in the package to register all tests you wish to make available
+// for kola to run.
+func init() {
+	harness.Tests = append(harness.Tests, []harness.Test{
+		Test{
+			Name:        "versionCheck",
```

Comment:
Lets name 'Name' practical as a global name, useful to the end user. Something like "EtcdVersionCheck" would work.